arch/x86/soc/apollo_lake/Kconfig.defconfig

```diff
@@ -0,0 +1,49 @@
+#
+# Kconfig - Apollo Lake SoC configuration options
+#
+# Copyright (c) 2014-2015 Wind River Systems, Inc.
```

Comment:
this is new file anyway, why do we need this copyright?

Reply:
The file was copied from the arch/x86/soc/atom. So need to preserve copyright.